recipes/icu/all/conanfile.py

```diff
@@ -201,22 +191,21 @@ def build(self):
 
     @property
     def _data_filename(self):
-        vtag = self.version.split(".")[0]
+        vtag = str(self.version.split(".")[0])
```

Comment:
```suggestion
        vtag = Version(self.version).major
```
It returns a string too.